changelogs/fragments/mysql_user-fix-versions-compatibilities-regression.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
Looks like the original bug only went into Ansible 2.8, so could you please remove this changelog

https://github.com/ansible/ansible/pull/45355

Reply:
Otherwise we are documenting something that no one would ever have seen (in a release version of Anisble)

Reply:
@gundalow I recently explicitly asked in #ansible-devel about changes which affect 2.8-only code and @abadger said they should also get a changelog.